OpenNefia.Content/CharaMake/CharaMakeFeatWindowLayer.cs

```diff
@@ -0,0 +1,96 @@
+﻿using OpenNefia.Content.Prototypes;
+using OpenNefia.Content.UI.Element;
+using OpenNefia.Core.Audio;
+using OpenNefia.Core.Locale;
+using OpenNefia.Core.Prototypes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenNefia.Content.CharaMake
+{
+    [Localize("Elona.CharaMake.FeatSelect")]
+    public class CharaMakeFeatWindowLayer : CharaMakeLayer
+    {
+        public const string ResultName = "feats";
+        private FeatWindow Window = default!;
+        private Dictionary<FeatPrototype, int> SelectedFeats = default!;
+        private int FeatCount;
+        public CharaMakeFeatWindowLayer()
+        {
+            
+        }
+
+        public override void Initialize(CharaMakeData args)
+        {
+            base.Initialize(args);
+            Reset();
+        }
+
+        private void Reset()
+        {
+            Window?.Dispose();
+            SelectedFeats = new Dictionary<FeatPrototype, int>();
+            FeatCount = 3;
+            if (Data.TryGetValue(CharaMakeRaceSelectLayer.ResultName, out RacePrototype? race))
+            {
+                foreach (var feat in race.BaseFeats)
+                    SelectedFeats[feat.Key.ResolvePrototype()] = feat.Value;
```

Comment:
You can just use the strong ID: `feat.Key.GetStrongID()`

Reply:
changed in b1407d8